components/files/__examples__/actions.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Files from '~/components/files';
+import File from '~/components/files/file';
+import MoreFiles from '~/components/files/more-files';
+import IconSettings from '~/components/icon-settings';
+import Icon from '~/components/icon';
+
+class Example extends React.Component {
+	static displayName = 'filesLoadingExample';
+
+	render() {
+		return (
+			<IconSettings iconPath="/assets/icons">
+				<Files id="files-with-actions-example">
+					<File
+						id="file-without-title"
+						image="/assets/images/placeholder-img@16x9.jpg"
+						onClickDownload={() => {
+							console.log('Download Button Clicked');
+						}}
+						onClickMoreActions={() => {
+							console.log('Download Button Clicked');
+						}}
+					/>
+					<File
+						id="file-with-title"
+						href="javascript:void(0);"
+						title="Image Title"
+						icon={<Icon category="doctype" name="pdf" />}
+						image="/assets/images/placeholder-img@16x9.jpg"
+						onClickDownload={() => {
+							console.log('Download Button Clicked');
+						}}
+						onClickMoreActions={() => {
```

Comment:
@aswinshenoy a callback here won't be sufficient since this is intended to be a dropdown. Change the `onClickMoreActions` prop to `moreActionsDropdown`, then make this example pass in a menu-dropdown component. Something like the following:
```
<Dropdown
	iconCategory="utility"
	iconName="down"
	iconVariant="border-filled"
	onSelect={(value) => {
		console.log('selected: ', value);
	}}
	options={[
		{ label: 'Menu Item One', value: 'A0' },
		{ label: 'Menu Item Two', value: 'B0' },
	]}
	value="A0"
/>
```
https://www.lightningdesignsystem.com/components/menus/#Base